interfaces/vrep/robots/YouBotVrepRobot.m

```diff
@@ -41,28 +41,21 @@
 % DQ Robotics website: dqrobotics.sourceforge.net
 %
 % Contributors to this file:
-%     Murilo Marques Marinho - murilo@nml.t.u-tokyo.ac.jp
+%     1. Murilo Marques Marinho - murilo@nml.t.u-tokyo.ac.jp
+%        - Responsible for the original implementation
+%     2. Frederico Fernandes Afonso Silva (frederico.silva@ieee.org)
+%        - Updated for compatibility with the DQ_SerialVrepRobot class.
 
-classdef YouBotVrepRobot < DQ_VrepRobot
-    
-    properties
-        joint_names;
-        base_frame_name;
-    end
-    
+classdef YouBotVrepRobot < DQ_SerialVrepRobot    
     properties (Constant)
         adjust = ((cos(pi/2) + DQ.i*sin(pi/2)) * (cos(pi/4) + DQ.j*sin(pi/4)))*(1+0.5*DQ.E*-0.1*DQ.k);
     end
     
     methods
-        function obj = YouBotVrepRobot(robot_name,vrep_interface)
-            %% Constructs an instance of a YouBotVrepRobot
-            %  >> vi = VrepInterface()
-            %  >> vi.connect('127.0.0.1',19997);
-            %  >> robot = YouBotVrepRobot("youBot", vi)
-            obj.robot_name = robot_name;
-            obj.vrep_interface = vrep_interface;
+        function obj = YouBotVrepRobot(robot_name, vrep_interface)
+            obj@DQ_SerialVrepRobot("youBot", 7, robot_name, vrep_interface);
             
+            %% youBot don't follow the standard name convention on CoppeliaSim. Also, the use of 'set_names()', as is done in the C++ implementation, is not supported on a constructor in MATLAB
```

Comment:
There are some ways to fix this but how about

> youBot` **does not** follow the standard **naming** convention **of/in** CoppeliaSim.

Reply:
Fixed in [fa35f1b](https://github.com/dqrobotics/matlab/pull/108/commits/fa35f1bd334ede9c8bd6aadb1fcc0d504edc1786).